tests/UI/PluginsActionsTest.php

```diff
@@ -1,47 +0,0 @@
-<?php
```

Comment:
Are we sure to delete this file?

Reply:
Yes, because it already exists at [`tests/Integrations/UI/PluginsActionsTest.php`](https://github.com/Parsely/wp-parsely/blob/develop/tests/Integration/UI/PluginsActionsTest.php).

I'd added it as a separate PR recently because I thought it was missing after I messed up a rebase, but I just wasn't looking for it in the right place.